src/hooks/chart.ts

```diff
@@ -817,6 +882,160 @@ const parseBarResponse = (trace: Trace, plot: Plot, responseData: ResponseData)
   return result;
 };
 
+
+/**
+ * Parses response data obtained for every trace within the plot
+ *
+ * @param trace current trace the plot config
+ * @param plot plot config
+ * @param responseData data received from request to be parsed
+ * @returns
+ */
+const parseHeatmapResponse = (trace: Trace, plot: Plot, responseData: ResponseData, additionalLayout: any) => {
```

Comment:
`additionalLayout` is not needed as a parameter. No contents should be defined on this object since this function is supposed to "create" that layout object and return it to `parsePlotData`.